api/buckets/service.go

```diff
@@ -1698,12 +1753,23 @@ func (s *Service) ArchiveWatch(req *pb.ArchiveWatchRequest, server pb.API_Archiv
 		return ErrArchivingFeatureDisabled
 	}
 
+	var ffsInfo *mdb.FFSInfo
+	if acct := accountFromContext(server.Context()); acct != nil {
+		ffsInfo = acct.FFSInfo
+	} else if user := userFromContext(server.Context()); user != nil {
+		ffsInfo = user.FFSInfo
+	}
+
+	if ffsInfo == nil {
+		return fmt.Errorf("no account or no FFS info associated with account")
+	}
```

Comment:
In `ArchiveWatch` we get the ffs info from the `Account` or `User` now.

Reply:
If there is no FFS info, just return that error. FFS instance is only lazily created on a call to `Archive`